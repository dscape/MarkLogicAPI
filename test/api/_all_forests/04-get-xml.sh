#!/bin/sh
curl -sH "Accept: application/xml" localhost:6984/_all_forests \
  | grep '>Documents</database>' | wc
