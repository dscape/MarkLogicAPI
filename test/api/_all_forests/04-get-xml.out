<forests>
  <forest id="10197055408561029936">Triggers</forest>
  <forest id="14016953481382448642">Last-Login</forest>
  <forest id="15413989367111532755">Schemas</forest>
  <forest id="4101250959913806120">Documents</forest>
  <forest id="15328071620698683416">App-Services</forest>
  <forest id="12620115732011072297">Security</forest>
  <forest id="5666773163079113644">Fab</forest>
  <forest id="8907563467982649407">Modules</forest>
</forests>
 